### DISCUSSION TITLE
Decide the purpose of the app

### DISCUSSION THREAD

**Discussion by danirabbit** [elementary/contacts#37] [General]
Possibly another spot for a user study!

What is the purpose of a contacts app in a modern desktop?

Add/Remove contacts?

Launch apps?

Track interactions with people?

Social features?

**Comment by henrique-andrade** [comment]
Dear Dani, I really didn't understand the *launch apps* purpose. Could you please explain a little bit? 🙂

**Comment by martinstork** [comment]
> Dear Dani, I really didn't understand the _launch apps_ purpose. Could you please explain a little bit? slightly_smiling_face

Launching apps based on the contact information:
Email address -> elementaryOS mail (compose mail window)
